cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlannerTest.java

```diff
@@ -218,11 +218,9 @@ public void testClear() {
                                           Collections.<String>emptySet());
 
     planner.addExecutionProposals(proposals, expectedCluster, null);
-    assertEquals(4, planner.remainingDataToMoveInMB());
     assertEquals(2, planner.remainingLeadershipMovements().size());
     assertEquals(2, planner.remainingReplicaMovements().size());
     planner.clear();
-    assertEquals(0, planner.remainingDataToMoveInMB());
```

Comment:
I am curious on whether we have any new test to verify `remainingDataToMoveInMB` after the refactoring?